lib/mini_racer/truffleruby.rb

```diff
@@ -0,0 +1,305 @@
+# frozen_string_literal: true
+
+require "monitor"
+
+module MiniRacer
+
+  class Context
+    def heap_stats
+      {
+        total_physical_size: 0,
+        total_heap_size_executable: 0,
+        total_heap_size: 0,
+        used_heap_size: 0,
+        heap_size_limit: 0,
+      }
+    end
+
+    def stop
+      if @context.respond_to?(:stop)
```

Comment:
Can be simplified now

Reply:
Simplified, I've removed this `if @context.respond_to?(:stop)`